cime_config/config_compsets.xml

```diff
@@ -104,6 +104,11 @@
 	<value compset="2013[CE]?_"     >2013-01-01</value>
       </values>
     </entry>
+    <entry id="LND_SETS_DUST_EMIS_DRV_FLDS">
+      <values match="first">
+      <value compset="_CAM7.*_CLM6">FALSE</value>
+      </values>
```

Comment:
Do we need a default value?  Have you tested with a cam6 clm5 compset?

Reply:
For CAM7/CLM6 compsets I'm getting the following error.
`Key: dust_emis_method, 
 Value 1: 'Zender_2003', 
 Value 2: 'Leung_2023'
ERROR: incompatible settings in drv_flds_in from 
`
Setting this default fixes the issue.

I've tested ERS.ne30pg3_t232.1850_CAM60_CLM50%BGC-CROP_CICE_MOM6_MOSART_DGLC%NOEVOLVE_SWAV.derecho_intel.allactive-defaultio, and there's no issue with dust emissions.

Here's the email I got from Erik about this.

> Hi Chris (adding Will)
>
> Thanks for pointing this out. This is an example of CAM and CTSM fighting over drv_flds_in settings.
>
> There is currently a XML variable that is intended to indicate which one has priority (LND_SETS_DUST_EMIS_DRV_FLDS).
>
> For CAM testing this is explicitly set to FALSE for CAM7 and CLM60 in config_compset.xml for CAM.
>
> See: https://github.com/ESCOMP/CAM/blob/cam_development/cime_config/config_compsets.xml#L902
>
> So I suggest you do the same thing in CESM for now.
>
> But, first just try running one of those cases with 
>
> LND_SETS_DUST_EMIS_DRV_FLD=FALSE
>
> And then bring in the CAM solution when you can show they work.
>
>
> This is an example of one of the glitchy things I'd like to get into CTSM and CESM as soon as we get a chance to. > That's one of the tags that we have on planned tags for CTSM. This is also the thing that I want to have a conversation around to have better long term solutions.

Reply:
I understand that this solves the problem for cam7/clm6 - what I want to know is if it has any negative consequences for other compsets.

Reply:
I'll do some more testing.

Reply:
I reran  ERS.ne30pg3_t232.1850_CAM60_CLM50%BGC-CROP_CICE_MOM6_MOSART_DGLC%NOEVOLVE_SWAV.derecho_intel.allactive-defaultio with this code change.  Ran fine, no answer changes.  This is the only fully active test that we run that uses CAM60 and CLM50.  This change shouldn't affect any fully active compsets that don't use cam7/clm6.

Reply:
Thank you for doing that.